src/shared/constants.ts

```diff
@@ -18,6 +18,7 @@ export const samAboutInstallUrl: string = 'https://aws.amazon.com/serverless/sam
 // tslint:disable-next-line:max-line-length
 export const vscodeMarketplaceUrl: string = 'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.aws-toolkit-vscode'
 export const githubUrl: string = 'https://github.com/aws/aws-toolkit-vscode'
+export const reportIssueUrl = `${githubUrl}/issues/new`
```

Comment:
If we're not prepopulating with information from VS Code, please prompt a user to use one of our existing templates by sending them to https://github.com/aws/aws-toolkit-vscode/issues/new/choose